reana_client/cli/files.py

```diff
@@ -67,10 +73,12 @@ def get_files(ctx, workflow, _filter,
     Examples: \n
     \t $ reana-client ls --workflow myanalysis.42
     """
+    JSON = 'json'
+    URL = 'url'
```

Comment:
We might need more output formats later, what do you think about moving it to config?